src/PrestaShopBundle/Resources/config/services/bundle/routing.yml

```diff
@@ -17,9 +17,21 @@ services:
     public: true
 
   prestashop.bundle.routing.converter.router_provider:
-    class: 'PrestaShopBundle\Routing\Converter\RouterProvider'
+    alias: PrestaShopBundle\Routing\Converter\RouterProvider
+    deprecated: ~
+
+  PrestaShopBundle\Routing\Converter\RouterProvider:
```

Comment:
I know you're eager to use autowiring, but I don't think it's been decided that we could yet 🤔

Reply:
Since this is local and not on the whole project it is fine and simplifies the configuration. Also it is [the recommended way](https://symfony.com/doc/4.4/contributing/code/standards.html#service-naming-conventions) to create services

Reply:
Recommended way by Symfony I never said otherwise, but not yet accepted by PrestaShop I personally don't mind but it's not a decision you can take alone 🤷‍♂️